src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -179,6 +214,130 @@ def create_webdriver(self, driver_name, alias=None, kwargs={}, **init_kwargs):
         self._debug("Created %s WebDriver instance with session id %s" % (driver_name, driver.session_id))
         return self._cache.register(driver, alias)
 
+    def save_webdriver(self, file=''):
+        """Stores the current web-driver session.
+
+        This KW makes the WebDriver sessions persistent. I.e. you can store the session(s) and leave the browser
```

Comment:
1) Write that this is only for purposed for test development and is not for production use. 

2) The documentation need some reformatting. But lets discuss that later.